swatch-product-configuration/src/test/java/com/redhat/swatch/configuration/registry/SubscriptionDefinitionTest.java

```diff
@@ -396,7 +412,7 @@ private static Stream<Arguments> ansibleTestCases() {
     var lookupParams2 =
         ProductTagLookupParams.builder()
             .metricIds(Set.of())
-            .engIds(Set.of("69", "204", "83", "408"))
+            .engIds(Set.of("69", "83", "408"))
```

Comment:
I had to remove this, otherwise an additional product is identified, apart from ansible. This is an expected (not sure if correct) change in behaviour caused by these changes.